### ISSUE TITLE
Opened Action Menu a11y tab behavior

### ISSUE THREAD

**Issue by simshanith** [adobe/spectrum-web-components#1488]
### Expected Behaviour

User can <kbd>tab</kbd> forward from an opened action menu to the next tabIndex.

### Actual Behaviour

When tabbing forward from opened & focused action menu, focus is thrown outside of document to browser controls instead of next element in document.

Able to <kbd>shift</kbd>+<kbd>tab</kbd> to go back as expected, but cannot advance forward from open state.

### Reproduce Scenario (including but not limited to)

On https://opensource.adobe.com/spectrum-web-components/components/action-menu tabbing from an example action menu button when closed advances to the "Copy to Clipboard" button. When open, goes to my browser tab outside of the document.

#### Steps to Reproduce

- Open action menu
- focus anywhere inside menu (focused by default on open)
- <kbd>tab</kbd>

**Comment by Westbrook** [comment]
Can you visit https://westbrook-picker-focus--spectrum-web-components.netlify.app/components/action-menu and confirm whether you're seeing the behavior expected here?

I'm thinking this is a good to go and good to get out, event if this experience gets altered more completely in #1424. However, I'm having a little trouble testing for a regression...

**Comment by Westbrook** [comment]
Looking deeper at the tests, I found the `sp-opened` is being dispatched earlier in the logical order of the experience (not after an animation if one is present) but that the `ActiveOverlay` was taking focus _after_ that animation, which ruins the test timing. I feel like this can easily cause flaky tests (as well as flaky functionality) and it seems like we should push that `sp-opened` event to _always_ be after the animation. Thoughts?

**Comment by simshanith** [comment]
> Can you visit https://westbrook-picker-focus--spectrum-web-components.netlify.app/components/action-menu and confirm whether you're seeing the behavior expected here?

Confirmed! That netlify link demonstrates the desired focus behavior.
